net-misc/trackma/trackma-0.7.3.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python{3_4,3_5} )
+
+inherit eutils distutils-r1
+
+SRC_URI="https://github.com/z411/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+KEYWORDS="~amd64"
+
+DESCRIPTION="Open multi-site list manager for media tracking sites"
+HOMEPAGE="https://github.com/z411/trackma"
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="inotify cli +urwid gtk qt5"
+
+DEPEND="urwid? ( dev-python/urwid[${PYTHON_USEDEP}] )
```

Comment:
Sort. Also, are they all really build-time dependencies? i.e. does the setup.py actually check for them or import modules that need them?